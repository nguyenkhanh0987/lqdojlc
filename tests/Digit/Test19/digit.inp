100
91871113 122281512
661301346 977078144
205969404 766243715
233677393 764235973
904798780 994870010
481650975 524920894
222911536 237317762
432976974 608058914
570677012 971927986
940683444 956160735
126144726 534996564
303951077 798615279
569505186 935179758
638101554 918205850
114417502 288345546
843783978 942052004
679816634 742624544
631975360 804019466
211972031 314398342
882419437 922257953
346805266 776686552
181524512 460097972
468060544 540990022
503047269 865225076
327832881 369267147
333595616 579634449
733260421 963943303
425828946 729862375
293853794 413753964
858387729 883252245
161786727 214205261
998547332 998632873
294308947 952700448
52334839 170707633
918799031 950510894
928591043 945783231
562737257 881323057
336172223 516398922
168104424 451504732
540069278 860828272
235320542 875773788
141306763 463836771
240114261 548223998
493445522 630215161
513585554 991980587
221334503 284589211
410505391 614327567
696164566 985427331
34350018 82495759
84967387 128581409
727705019 889664320
717458662 909096826
181712589 827114059
462864273 601320597
251544632 256616322
980383370 987214260
569311438 901826188
458573548 665481315
271665673 604079047
15084962 824715242
66475713 524705978
591076567 838426834
433087368 678476565
16828272 192434280
214882672 513216606
97558968 307061538
186665989 195330385
498511064 994359092
489237954 512796060
612922645 904227449
229462817 327405801
206994968 991714580
564259671 966516458
33461441 150439271
500954873 506411490
899780872 922829926
355478785 530167587
742417624 864672991
43641743 136944911
60667584 304643506
607084334 962733717
162262573 289980519
277906150 959938767
718123060 802938025
321323521 602532044
812458964 870937191
72867058 104060130
127589482 766081331
199165221 326655053
88239172 331247586
851689618 926581522
320668036 518957451
519088184 524436791
504883673 727708542
352923761 904534762
574789583 956585282
181187106 835044251
753692676 784412302
66802177 708542368
806572050 927290217
