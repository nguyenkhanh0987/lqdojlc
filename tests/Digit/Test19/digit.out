5045648
25387967
71922933
68544225
5465165
5409441
1888885
23127344
34591433
923429
64644712
59206084
32619622
24374874
31824547
6774632
5520689
17320686
17377888
2709792
51474443
43569335
9117923
33786588
6123561
33727449
17028189
34802133
17881200
2574536
11017633
5451
70136765
23879928
1177625
772481
30057260
26415020
45623485
30847998
77328077
53073627
45431609
14580282
42339040
11433695
25548402
20868035
10883814
7696430
13373253
15449582
82784389
19002571
1024840
867435
31729512
24670120
47960944
115956848
76084267
21921161
29402215
35877580
45581537
37757772
2334551
43738975
3092061
25620839
16724073
88699104
34912391
23198191
308595
562291
24442870
9907880
19041240
47507702
28225813
24073097
74486028
8155014
39272950
3375461
8005925
88318262
19975039
42706561
5734385
28404537
502083
22806904
61183310
32607511
83353902
3515345
96776489
7706342
